core/validator.go

```diff
@@ -87,12 +88,20 @@ func (v *shardValidator) UpdateOperatorID(operatorID OperatorID) {
 }
 
 func (v *shardValidator) ValidateBatch(batchHeader *BatchHeader, blobs []*BlobMessage, operatorState *OperatorState, pool common.WorkerPool) error {
-
-	err := validateBatchHeaderRoot(batchHeader, blobs)
+	headers := make([]*BlobHeader, len(blobs))
```

Comment:
Are there limits to the number of blobs that can be requested ?

Reply:
There is a limit on the size of the payload but not on the number of blobs